src/components/Message/MessageSimple.tsx

```diff
@@ -187,7 +188,7 @@ const MessageSimpleWithContext = <
               {message.attachments?.length && !message.quoted_message ? (
                 <Attachment actionHandler={handleAction} attachments={message.attachments} />
               ) : null}
-              {message.ai_generated ? (
+              {isMessageAIGenerated(message) ? (
```

Comment:
Could be probably run in a single place as we do not know, what the integrators will put there.

Reply:
You mean something like calculating `isAIGenerated` for example before passing it to the `MessageContext` ?